Mustache/src/js/main.js

```diff
@@ -0,0 +1,96 @@
+//const Mustache = require('mustache');
+
+var data = {};
+
+function validar(e) {
+    tecla = (document.all) ? e.keyCode : e.which;
+    if (tecla===13) callPromise();
+}
+
+function promiseC() {
+    return new Promise(function(resolve, reject) {
```

Comment:
Use arrow functions instead
`return new Promise((resolve, reject) => { ... })`

https://medium.freecodecamp.org/when-and-why-you-should-use-es6-arrow-functions-and-when-you-shouldnt-3d851d7f0b26